src/app/scheduler/components/SchedulerCalendar.tsx

```diff
@@ -28,14 +31,61 @@ const localizer = dateFnsLocalizer({
   locales,
 });
 
+const slotPropGetter = (date: Date, resourceId?: number | string) => {
+  const today = new Date();
+  if (
+    date.getFullYear() === today.getFullYear() &&
+    date.getMonth() === today.getMonth() &&
+    date.getDate() === today.getDate()
```

Comment:
I've got to think there's something in DateFNS that simplifies this.

Reply:
if not, maybe some helper functions regarding dates would be useful

Reply:
^ moved to util function in shared/dates.ts